test/spec_helper.rb

```diff
@@ -27,7 +27,7 @@
 #
 module Common
   at_exit do
-    exit! 0
+    @@autorun = false
```

Comment:
this is no longer needed right?

Reply:
its actually an old bug from when i first wrote that spec_helper.rb.  with the `exit! 0` it will "hide" some failed tests. The new line `@@autorun = false` means when you create the new check it won't auto run now.